package.json

```diff
@@ -46,24 +46,24 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "@cucumber/cucumber": "^8.3.0",
-    "@cucumber/cucumber-expressions": "^15.2.0",
+    "@cucumber/cucumber": "^8.5.1",
+    "@cucumber/cucumber-expressions": "^16.0.0",
```

Comment:
As the point of this commit is to get the package compatible with TestCafe@1.20.0, let's keep major versions of other packages unchanged for now.
Revert to 15.2.0